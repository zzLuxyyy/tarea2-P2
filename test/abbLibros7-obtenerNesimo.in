crearABBLibrosVacio
crearFecha 01/05/1992
crearLibro 140003446 "Jazz" "Toni" "Morrison" "Una historia de amor y crimen en el Harlem de los años 20" 3
insertarLibroABBLibros
obtenerNesimoLibroABBLibros 1
crearFecha 01/01/1971
crearLibro 204578912 "Las venas abiertas de América Latina" "Eduardo" "Galeano" "Un ensayo crítico sobre la historia de la explotación en América Latina" 7
insertarLibroABBLibros
obtenerNesimoLibroABBLibros 1
obtenerNesimoLibroABBLibros 2
crearFecha 16/07/1951
crearLibro 316769177 "El guardián en el centeno" "J.D." "Salinger" "Holden narra sus días tras ser expulsado del colegio, en busca de sentido" 1
insertarLibroABBLibros
crearFecha 01/09/2005
crearLibro 999001122 "Nunca me abandones" "Kazuo" "Ishiguro" "Jóvenes clonados enfrentan su destino en un mundo distópico" 2
insertarLibroABBLibros
crearFecha 26/06/1997
crearLibro 171819202 "Harry Potter y la piedra filosofal" "J.K." "Rowling" "Un niño mago descubre su destino en la escuela Hogwarts" 6
insertarLibroABBLibros
crearFecha 01/10/1995
crearLibro 843397030 "Ensayo sobre la ceguera" "José" "Saramago" "Una epidemia de ceguera azota a una ciudad, revelando lo peor del ser humano" 2
insertarLibroABBLibros
crearFecha 01/01/2002
crearLibro 849550144 "Juventud" "J.M." "Coetzee" "Un joven sudafricano busca sentido en su vida y vocación en Londres" 4
insertarLibroABBLibros
crearFecha 01/01/2008
crearLibro 880619315 "La soledad de los números primos" "Paolo" "Giordano" "Dos almas heridas incapaces de coincidir en el amor" 1
insertarLibroABBLibros
crearFecha 28/01/1813
crearLibro 111213141 "Orgullo y prejuicio" "Jane" "Austen" "Elizabeth Bennet enfrenta el amor y los prejuicios de su época" 1
insertarLibroABBLibros
crearFecha 08/06/1949
crearLibro 123456789 "1984" "George" "Orwell" "Un régimen totalitario controla hasta los pensamientos de sus ciudadanos" 2
insertarLibroABBLibros
obtenerNesimoLibroABBLibros 10
obtenerNesimoLibroABBLibros 9
obtenerNesimoLibroABBLibros 8
obtenerNesimoLibroABBLibros 7
obtenerNesimoLibroABBLibros 6
obtenerNesimoLibroABBLibros 1
obtenerNesimoLibroABBLibros 2
obtenerNesimoLibroABBLibros 3
obtenerNesimoLibroABBLibros 4
obtenerNesimoLibroABBLibros 5
liberarABBLibros
Fin
