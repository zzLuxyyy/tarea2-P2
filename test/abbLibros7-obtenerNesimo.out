1>Se ejecutó crearTABBLibrosVacio exitosamente.
2>3>4>Se insertó el libro con ISBN 140003446 al árbol.
5>Libro nro 1 del abb libros:
Libro 140003446
Título: Jazz
Autor: Toni Morrison
Descripción: Una historia de amor y crimen en el Harlem de los años 20
Género: 3
Fecha de edición: 1/5/1992
6>7>8>Se insertó el libro con ISBN 204578912 al árbol.
9>Libro nro 1 del abb libros:
Libro 140003446
Título: Jazz
Autor: Toni Morrison
Descripción: Una historia de amor y crimen en el Harlem de los años 20
Género: 3
Fecha de edición: 1/5/1992
10>Libro nro 2 del abb libros:
Libro 204578912
Título: Las venas abiertas de América Latina
Autor: Eduardo Galeano
Descripción: Un ensayo crítico sobre la historia de la explotación en América Latina
Género: 7
Fecha de edición: 1/1/1971
11>12>13>Se insertó el libro con ISBN 316769177 al árbol.
14>15>16>Se insertó el libro con ISBN 999001122 al árbol.
17>18>19>Se insertó el libro con ISBN 171819202 al árbol.
20>21>22>Se insertó el libro con ISBN 843397030 al árbol.
23>24>25>Se insertó el libro con ISBN 849550144 al árbol.
26>27>28>Se insertó el libro con ISBN 880619315 al árbol.
29>30>31>Se insertó el libro con ISBN 111213141 al árbol.
32>33>34>Se insertó el libro con ISBN 123456789 al árbol.
35>Libro nro 10 del abb libros:
Libro 999001122
Título: Nunca me abandones
Autor: Kazuo Ishiguro
Descripción: Jóvenes clonados enfrentan su destino en un mundo distópico
Género: 2
Fecha de edición: 1/9/2005
36>Libro nro 9 del abb libros:
Libro 880619315
Título: La soledad de los números primos
Autor: Paolo Giordano
Descripción: Dos almas heridas incapaces de coincidir en el amor
Género: 1
Fecha de edición: 1/1/2008
37>Libro nro 8 del abb libros:
Libro 849550144
Título: Juventud
Autor: J.M. Coetzee
Descripción: Un joven sudafricano busca sentido en su vida y vocación en Londres
Género: 4
Fecha de edición: 1/1/2002
38>Libro nro 7 del abb libros:
Libro 843397030
Título: Ensayo sobre la ceguera
Autor: José Saramago
Descripción: Una epidemia de ceguera azota a una ciudad, revelando lo peor del ser humano
Género: 2
Fecha de edición: 1/10/1995
39>Libro nro 6 del abb libros:
Libro 316769177
Título: El guardián en el centeno
Autor: J.D. Salinger
Descripción: Holden narra sus días tras ser expulsado del colegio, en busca de sentido
Género: 1
Fecha de edición: 16/7/1951
40>Libro nro 1 del abb libros:
Libro 111213141
Título: Orgullo y prejuicio
Autor: Jane Austen
Descripción: Elizabeth Bennet enfrenta el amor y los prejuicios de su época
Género: 1
Fecha de edición: 28/1/1813
41>Libro nro 2 del abb libros:
Libro 123456789
Título: 1984
Autor: George Orwell
Descripción: Un régimen totalitario controla hasta los pensamientos de sus ciudadanos
Género: 2
Fecha de edición: 8/6/1949
42>Libro nro 3 del abb libros:
Libro 140003446
Título: Jazz
Autor: Toni Morrison
Descripción: Una historia de amor y crimen en el Harlem de los años 20
Género: 3
Fecha de edición: 1/5/1992
43>Libro nro 4 del abb libros:
Libro 171819202
Título: Harry Potter y la piedra filosofal
Autor: J.K. Rowling
Descripción: Un niño mago descubre su destino en la escuela Hogwarts
Género: 6
Fecha de edición: 26/6/1997
44>Libro nro 5 del abb libros:
Libro 204578912
Título: Las venas abiertas de América Latina
Autor: Eduardo Galeano
Descripción: Un ensayo crítico sobre la historia de la explotación en América Latina
Género: 7
Fecha de edición: 1/1/1971
45>46>Fin.
